snmp/datadog_checks/snmp/data/default_profiles/apc-netbotz.yaml

```
@@ -352,4 +352,4 @@ metrics:
         mapping:
           0: no
           1: yes
-          2: none
+          2: null
```

```suggestion
          2: "null"
```

Maybe this ☝️ might work better since `null` seems to lead to empty string: https://github.com/DataDog/integrations-core/pull/15668/files#r1307606282

---

I think we want the string "null" since that's the mapping string:

```
BoolValue ::= TEXTUAL-CONVENTION
    STATUS current
    DESCRIPTION
        "A boolean value."
    SYNTAX INTEGER { no(0), yes(1), null(2) }
```

I think we should update the tooling to always put quotes around the mapping. Working on this.

---

Strange, when I run `ndmprofiles update-profile` I'm getting:

```
      - column:
          OID: 1.3.6.1.4.1.5528.100.5.1.1.6
          name: errorCondResolved
        tag: netbotz_error_cond_resolved
        mapping:
          0: no
          1: yes
          2: 'null'
```

with single quote around `'null'`.

---

not sure
addressed it in [e20acd7](https://github.com/DataDog/integrations-core/pull/15668/commits/e20acd7329125049754b413c2f78fccb81e9b76e)